### PR TITLE
updating links in help module

### PR DESCRIPTION
**Pull request by rybmat**
https://wikia-inc.atlassian.net/browse/WW-263

@Wikia/west-wing

### CODE REVIEW

File: extensions/wikia/CommunityPage/templates/helpModule.mustache [hunk 1]
```diff
@@ -2,5 +2,6 @@
 	<h2>{{helpModule.title}}</h2>
 	<a data-tracking="help-page-edit-page-link" href="{{helpModule.editPageLink}}">{{helpModule.editPage}}&nbsp;&rarr;</a><br/>
 	<a data-tracking="help-page-add-links-page-link" href="{{helpModule.addLinksPageLink}}">{{helpModule.addLinks}}&nbsp;&rarr;</a><br/>
-	<a data-tracking="help-page-add-new-page-link" href="{{helpModule.addNewPageLink}}">{{helpModule.addNewPage}}&nbsp;&rarr;</a>
+	<a data-tracking="help-page-add-new-page-link" href="{{helpModule.addNewPageLink}}">{{helpModule.addNewPage}}&nbsp;&rarr;</a><br/>
+    <a data-tracking="help-page-policy-link" href="{{helpModule.communityPolicyLink}}">{{helpModule.communityPolicy}}&nbsp;&rarr;</a>
```
Comment (sebastianmarzjan) [inline extensions/wikia/CommunityPage/templates/helpModule.mustache]: Minor: spacing

**Comment by sebastianmarzjan** [comment]
one minor comment, lgtm

File: extensions/wikia/CommunityPage/templates/helpModule.mustache [hunk 2]
```diff
@@ -2,5 +2,6 @@
 	<h2>{{helpModule.title}}</h2>
 	<a data-tracking="help-page-edit-page-link" href="{{helpModule.editPageLink}}">{{helpModule.editPage}}&nbsp;&rarr;</a><br/>
 	<a data-tracking="help-page-add-links-page-link" href="{{helpModule.addLinksPageLink}}">{{helpModule.addLinks}}&nbsp;&rarr;</a><br/>
-	<a data-tracking="help-page-add-new-page-link" href="{{helpModule.addNewPageLink}}">{{helpModule.addNewPage}}&nbsp;&rarr;</a>
+	<a data-tracking="help-page-add-new-page-link" href="{{helpModule.addNewPageLink}}">{{helpModule.addNewPage}}&nbsp;&rarr;</a><br/>
+	<a data-tracking="help-page-policy-link" href="{{helpModule.communityPolicyLink}}">{{helpModule.communityPolicy}}&nbsp;&rarr;</a>
```
Comment (lukaszkonieczny) [inline extensions/wikia/CommunityPage/templates/helpModule.mustache:6]: What do you think about checking if link is not empty?

Comment (rybmat) [reply extensions/wikia/CommunityPage/templates/helpModule.mustache:6]: talked to Trevor, he'd like still to display the link. I asked him where it should point if empty but haven't got answer yet

**Comment by lukaszkonieczny** [comment]
LGTM

File: extensions/wikia/CommunityPage/templates/helpModule.mustache (hunk 2 above)
Comment (rybmat) [reply extensions/wikia/CommunityPage/templates/helpModule.mustache:6]: WW-276

**Pull request merged**